static/src/javascripts/projects/common/modules/commercial/adblock-rules.js

```diff
@@ -0,0 +1,30 @@
+define([
```

Comment:
Perhaps this should just concern whether the message is displayed? It could just return a function,

    hasAdblockMessages() {Boolean}

You could call that in donot-use-adblock and then your test in sticky.js would be -

    if (hasAdblockMessages()) {
        // do DOM manipulation...
    }


    getUpdateMethod = function() {
        if (detect.hasAdblock() && !hasAdblockMessages(){ {
              // this code doesn't actually care if someone is a member. It just cares whether something
              // is going to replace the upper banner.
        }
    }